dpgen/generator/run.py

```diff
@@ -2190,8 +2190,11 @@ def _trust_limitation_check(sys_idx, lim):
             cwd = os.getcwd()
             os.chdir(fp_task_path)
             if cluster_cutoff is None:
-                if model_devi_engine in ("lammps", "gromacs"):
-                    os.symlink(os.path.relpath(conf_name), 'conf.dump')
+                if model_devi_engine in ("lammps", "gromacs"):                   
+                    if model_devi_merge_traj:
+                        conf_sys.to("lammps/lmp", "conf.dump")
```

Comment:
I see line 2193 has `gromacs`, but I think this key only applies to `lammps`?

Reply:
Yes, it only applies to `lammps`. Since `gromacs` shares the traj format `lammps/dump`, see github.com/deepmodeling/dpgen/blob/devel/dpgen/generator/run.py#L2148-L2150, they are treated in the same way here. But `merge_traj` don't support `gromacs`, so I will divide them.